src/java.base/share/classes/java/util/concurrent/ConcurrentHashMap.java

```diff
@@ -351,7 +354,7 @@
      * Actual hash code distributions encountered in practice
      * sometimes deviate significantly from uniform randomness.  This
      * includes the case when N > (1<<30), so some keys MUST collide.
-     * Similarly for dumb or hostile usages in which multiple keys are
+     * Similarly, for dumb or hostile usages in which multiple keys are
```

Comment:
"Similarly" should almost always be followed by a comma, but this is an exception.